moveit_py/src/moveit/moveit_core/robot_model/joint_model_group.cpp

```diff
@@ -59,13 +59,16 @@ void init_joint_model_group(py::module& m)
                     str: The name of the joint model group.
                     )")
 
+      .def_property_readonly("link_model_names", &moveit::core::JointModelGroup::getLinkModelNames)
       .def_property("joint_model_names", &moveit::core::JointModelGroup::getJointModelNames, nullptr,
                     R"(
                     list[str]: The names of the joint models in the group.
                     )")
       .def_property("active_joint_model_names", &moveit::core::JointModelGroup::getActiveJointModelNames, nullptr)
       .def_property("active_joint_model_bounds", &moveit::core::JointModelGroup::getActiveJointModelsBounds, nullptr,
                     py::return_value_policy::reference_internal)
+      .def("get_only_one_end_effector_tip",
+           [](const moveit::core::JointModelGroup* self) { return self->getOnlyOneEndEffectorTip()->getName(); })
```

Comment:
do we need a check if we have an only eef tip? Also, this could be a property just like the other string getters above